puck.config.tsx

```diff
@@ -261,6 +263,22 @@ export const config = {
       },
       render: Code as any,
     },
+    Embed: {
+      fields: {
+        height: { type: "number" },
+        src: { type: "text" },
+        title: { type: "text" },
+        width: { type: "number" },
+      },
+
+      defaultProps: {
+        height: 100,
+        src: "https://google.com",
```

Comment:
**praise:** Does Google actually load in an iframe? I think they block it.

Reply:
Yeah, they did.
Will correct that.